cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/auth/OAuth2TokenProvider.java

```diff
@@ -8,4 +8,6 @@ public interface OAuth2TokenProvider {
 
     OAuth2Token refreshToken() throws IOException;
 
+    OAuth2Token getDirectX509Token() throws IOException;
```

Comment:
Do we care if this is X509? Also, do we expect this to be a new instance? If so, I'd think renaming to `newToken()` would be more appropriate. That would distinguish from `refreshToken()`.

Reply:
No we don't care. I updated to newToken()